python/src/trezorlib/cli/settings.py

```diff
@@ -190,9 +190,12 @@ def homescreen(client, filename):
 
 
 @cli.command()
+@click.option(
+    "-t", "--temporary", is_flag=True, help="Revert setting after unplugging Trezor."
+)
 @click.argument("level", type=ChoiceType(SAFETY_LEVELS))
 @with_client
-def safety_checks(client, level):
+def safety_checks(client, temporary, level):
```

Comment:
please add a brief note about this to the docstring (which is shown on `set safety-checks --help`)